doc/authoring_command_modules/authoring_commands.md

```diff
@@ -1,5 +1,10 @@
 The document provides instructions and guidelines on how to author individual commands.
 
+> **Note**
+> 
+> [aaz-dev-tools](https://azure.github.io/aaz-dev-tools/) is our codegen tool which can be used for Control Plane APIs. Highly recommend using it in new commands.
```

Comment:
```suggestion
> [aaz-dev-tools](https://azure.github.io/aaz-dev-tools/) is our codegen tool which can be used for Control Plane APIs. Highly recommend using it in new commands. It brings the following benefits:
> - Shorten development cycle (save 50%+ development time).
> - Consistent code style improves command quality and saves PR review time.
> - Directly generate CLI commands from swagger instead of waiting for SDK release.
```